CFEScraping/index.js

```diff
@@ -0,0 +1,13 @@
+const puppeteer = require('puppeteer');
+
+(async () => {
+    const browser = await puppeteer.launch();
+    const page = await browser.newPage();
+
+    await page.goto('https://msc.cfe.mx/Aplicaciones/NCFE/Concursos/');
+    console.log('Successful');
+    await page.waitForTimeout(5000);
```

Comment:
5s maybe is too long how much take the page to load all the resources(the resources we need to manipulate, buttons inputs, etc) you can check this with your browser dev tools network section?